### DISCUSSION TITLE
Passing through input that doesn't match

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [junegunn/fzf#2493] [Q&A]
Is it possible to pass through an input without it matching anything?
For example in Dmenu when I'm passing let's say random characters, they're obviously won't be a match but the random characters are still printed out.
I'm looking for the same functionality in fzf.
( for a man pager script I'm running, not all manpages are visible by apropos so i want to still type the name of the page I'm looking for and still have it pass through my script)

**Comment by Lockszmith-GH** [comment]
did you ever resolve this?

**Comment by Delapouite** [comment]
Could the `--print-query` flag be what you're looking for?

**Reply by Lockszmith-GH** [reply]
Yes, not the OP, but this looks like what I was looking for.

I wrote a script to edit some commonly used files, but I also want it to be able to create new-files if needed.
`--print-query` solves this for me, if I end the query with a special character (let's say !), it will take the query instead of the result (or no-result).

Thank you so much!
